book/src/requirements.md

```diff
@@ -7,16 +7,13 @@ This page lists the requirements for running `bindgen` and how to get them.
 `bindgen` leverages `libclang` to preprocess, parse, and type check C and C++
 header files.
 
-It is recommended to use Clang 3.9 or greater, however `bindgen` can run with
+It is recommended to use Clang 4.0 or greater, however `bindgen` can run with
```

Comment:
@emilio, is it correct and helpful to continue to reference older-than-4.0 versions of Clang here ?

Reply:
Probably not much, also pretty much everyone should have newer clang at this point.